ui/app/router.js

```diff
@@ -210,6 +210,17 @@ Router.map(function () {
       this.route('replication-dr-promote', function () {
         this.route('details');
       });
+      // TODO remove conditional once further feature work for single item recovery for release 1.21 is completed
+      if (config.environment !== 'production') {
+        this.route('recovery', function () {
+          this.route('snapshots', function () {
+            this.route('load');
+            this.route('snapshot', function () {
+              this.route('manage');
+            });
```

Comment:
In the RFC we had talked about this segment of the URL being a dynamic param called `snapshot_id` (As an aside, it's a good practice to have the url param - `snapshot_id` - match the param we'll use from the API.) 

In practice, this means that buttons or actions that navigate to manage a snapshot would look something like:
```js
const snapshot_id = 1234
this.router.transitionTo('recovery.snapshots.snapshot', snapshot_id)
```

If this is still the plan we'll need to define the `path` ([docs](https://guides.emberjs.com/release/routing/defining-your-routes/#toc_dynamic-segments)) for the dynamic segment here. I'm also wondering if `manage` is even necessary 🤔 Part of me thinks we should just get rid of the child route since right now it's just a single view to either read or recover items. We could have the "read" view just live at `snapshot`. If doing more things with a snapshot becomes a thing we can always extend it in the future. What do you think?


```suggestion
            this.route('snapshot', { path: '/:snapshot_id } )
```